drivers/si70xx/si70xx.c

```diff
@@ -56,22 +57,22 @@ int si70xx_test(const si70xx_t *dev)
 
     uint8_t id = si70xx_get_id(dev);
 
-#if defined(MODULE_SI70XX_ID_SI7006)
+#if defined(MODULE_SI7006)
```

Comment:
Please move this if/else into si70xx_internals.h, like:

```
#ifdef MODULE_SI7006
#define SI70XX_ID SI70XX_ID_SI7006
#elif ...
...
#else
#error "Don't know SI70XX variant!"
#endif
```

Then just use one if in this function, comparing to ```SI70XX_ID```.